### PR TITLE
chore: snaps-cli 3.0.0

### PR DESCRIPTION
**Pull request by irubido**

### CODE REVIEW

**Comment by Rachale43** [comment]
Nope

**Review by Rachale43 — APPROVED** [review]

**Review by Rachale43 — CHANGES_REQUESTED** [review]
[]()

**Comment by Rachale43** [comment]
> []()

- [ ] -

File: packages/snap/snap.config.js [hunk 1]
```diff
@@ -1,8 +0,0 @@
-module.exports = {
-  cliOptions: {
-    src: './build/index.js',
-    dist: 'dist',
-    port: 8081,
-    outfileName: "bundle.js",
```
Comment (Rachale43) [inline packages/snap/snap.config.js:6]: No

**Comment by Lykhoyda** [comment]
@irubido I think it could be closed as the CLI was already updated in another PR

**Pull request closed without merging**